tests/resources/configs/simple_with_control.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v0.1
+workloads:
+  simple:
+    runtime: podman
+    restartPolicy: NEVER
+    agent: agent_A
+    runtimeConfig: |
+      image: ghcr.io/eclipse-ankaios/tests/alpine:latest
+      commandArgs: [ "echo", "Hello Ankaios"]
+    controlInterfaceAccess:
+      allowRules:
+        - type: StateRule
+          operation: Write
+          filterMask:
+            # workload has write access to the config of the workload dynamic_nginx
+            - "desiredState.workloads.dynamic_nginx"
```

Comment:
The rules are very specific and perhaps taken from the examples. Shall we change that to something more generic? The dynamic created nginx is only covered by the control interface examples.

Reply:
For now this is not needed. I would leave it for the next one needing it.

Reply:
Ok we can keep it.